app/templates/views/index.html

```diff
@@ -8,7 +8,13 @@ <h1 class="heading-large">Download your document</h1>
   </p>
   <p>
     If you are unsure if this is a genuine communication,
-    <a href={{ service_contact_link }}> contact {{ service_name }}</a>
+    {% if contact_info_type == "link" %}
+      <a href={{ service_contact_info }}> contact {{ service_name }}</a>
+    {% elif contact_info_type == "email" %}
+      <a href={{ "mailto:" + service_contact_info }}> contact {{ service_name }}</a>
```

Comment:
It would be better to also show the email address here. Not everyone has their email client properly configured to open `mailto` links.

Something like
> Contact Service Name by emailing [test@example.com](mailto:test@example.com)

Reply:
Good call, I will change it like so.